mempool/reactor.go

```diff
@@ -95,6 +100,11 @@ func (memR *Reactor) Receive(e p2p.Envelope) {
 	memR.Logger.Debug("Receive", "src", e.Src, "chId", e.ChannelID, "msg", e.Message)
 	switch msg := e.Message.(type) {
 	case *protomem.Txs:
+		if memR.WaitSync() {
+			memR.Logger.Info("ignore message received during syncing", "msg", msg)
```

Comment:
If syncing takes a while, we may have a large number of such messages in the log, in INFO level.

Reply:
Right. I will change it to debug level. The number of log messages will still be high, but I think they will be useful for debugging or testing.